File: LLD/InterviewExamples/Blinkit/Java/src/models/Warehouse.java
package models;

import java.util.UUID;
public class Warehouse {

    private String id;
    private String city;

    public Warehouse(String city) {
        id = UUID.randomUUID().toString();
        this.city = city;
    }

    public String getCity() {
        return city;
    }

    public String getId() {
        return id;
    }
}

File: LLD/InterviewExamples/Blinkit/Node/src/models/warehouse.js
const { randomUUID } = require('crypto')

class Warehouse {
    #id
    #city
    #inventory

    constructor(city,inventory) {
        this.#id = randomUUID();
        this.#city = city;
        this.#inventory = inventory;
    }

    getCity() {
        return this.#city;
    }

    getInventory() {
        return this.#inventory
    }
}